knowledge_repo/app/utils/posts.py

```diff
@@ -44,12 +44,9 @@ def get_posts(feed_params):
     if filters and str(filters):
         filter_set = get_query_param_set(filters)
         for elem in filter_set:
-            query = query.filter(or_(Post.title.like('%' + elem + '%'),
-                                     Post.title.like('%' + elem),
-                                     Post.title.like(elem + '%'),
-                                     Post.tldr.like('%' + elem + '%'),
-                                     Post.tldr.like('%' + elem),
-                                     Post.tldr.like(elem + '%')))
+            query = query.filter(or_(func.lower(Post.keywords).like('%' + elem + '%'),
```

Comment:
It didn't match empty string when we wrote this up originally. Not sure if our sqlalchemy version changed